### ISSUE TITLE
proper documentation of error messages on the API doc page

### ISSUE THREAD

**Issue by justadreamer** [51Degrees/cloud-issues#7]
# Background

The cloud API documentation page https://cloud.51degrees.com/api-docs/index.html contains a list of the errors with causes (response codes 400-429) for the .json API, but fails to provide the examples of the exact json responses for these errors. 

An example of the exact json response body for the cases of 400-429 errors would help to build a more robust app and test code. 

# Objective

Please illustrate the API doc with the json snippets of error response bodies.

**Comment by Steve51D** [comment]
Hi @justadreamer,

You're correct about this shortcoming of our documentation. We had a previous issue for this: #4.

However, we were unable to find a simple and satisfactory way to address it. Eventually, we decided to leave it as is for now.
It's been almost a year since then, so I'll raise this with the team again to see if we can do anything about it.

**Comment by justadreamer** [comment]
Hi @Steve51D!  Any ETA on this?  We depend on the error responses for developing and covering our library code with tests.  Thank you.

**Comment by Steve51D** [comment]
Hi @justadreamer,

I'm going to be making some other cloud changes this week, so will take a look whether Swashbuckle supports this yet. If so, I should be able to make the necessary changes.
Will update here once I know more.

**Comment by justadreamer** [comment]
Thank you @Steve51D.  

Another idea that could really help us with writing integration tests would be to add a param to the API call to simulate various error responses.  F.e. both JS and JSON API resources could accept a `simulateerrorcode` param where we would pass the error code we would want it to respond with: 

i.e. 
```http
GET https://cloud.51degrees.com/api/v4/<resource-key>.json?simulateerrorcode=429
```

How feasible would this be?

**Comment by Steve51D** [comment]
Hi @justadreamer,

I'm afraid we've decided not to allocate any resources to this in the near term. It's still something we want to address, we just don't have the capacity to do so right now.

**Comment by ben51degrees** [comment]
Closed with version 4.3.74
